### PR TITLE
prep v1.1.2

### PR DESCRIPTION
**Pull request by SebastienPeillet**
@pblottiere

### CODE REVIEW

**Comment by pblottiere** [comment]
Nice thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+- `nFollower` field on followers table
+
+-----
+
+## [v1.1.1] - 2021-12-20
+
+### Modified
+
+- patches the v1.1.0 to avoid typing error for python version 3.8
```
Comment (pblottiere) [inline CHANGELOG.md:45]:
```suggestion
- Patches the v1.1.0 to avoid typing error for python version 3.8
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+- Add keyboard shortcut
+
+
+### Modified
+
+- Translation fixes
+- Layer attribute constraints #67 #68 #69
+- Remove side attribute values in the sighting layer form
+- SAMMO-Boat log file will always be written in the plugin root folder
+- mode debug
```
Comment (pblottiere) [inline CHANGELOG.md:71]:
```suggestion
- Debug mode
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+- Update README.md with main features
+- By default, table are sorted by time desc
+- Configure `comment` fields without multiline to ease navigation with `Tab`
+- Remove almost all default values
+- Audio record increase to 1 min (sigthings / environment)
+- Remove duplication of the last entity for sightings (except `side` field)
+- `angle` field restriction on sighting table
+- `direction` field can be null on sighting table
+- rectrition of one value between `behavMaM`/`behavBirds`/`behavShip` fields on sighting table
```
Comment (pblottiere) [inline CHANGELOG.md:30]:
```suggestion
- Restriction of one value between `behavMaM`/`behavBirds`/`behavShip` fields on sighting table
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+- Debug mode
+
+
+-----
+
+## [v1.0.0] - 2021-10-08
+
+### Add
+
+
```
Comment (SebastienPeillet) [inline CHANGELOG.md:80]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+### Modified
+
+- Translation fixes
+- Layer attribute constraints #67 #68 #69
+- Remove side attribute values in the sighting layer form
+- SAMMO-Boat log file will always be written in the plugin root folder
+- Debug mode
+
+
```
Comment (SebastienPeillet) [inline CHANGELOG.md:73]:
```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+### Add
+
+- Audio dialog to play record
+- On adding a entity, the focus is automatically setted on the table line
+- Sightings / environment table can be resized independently
+- Undo / redo mechanisms
+- Export : sightings layer is exported with joined fields of species table
+- Export : environment layer is exported with joined fields of observer table (for left/right/center observer)
+
+
```
Comment (SebastienPeillet) [inline CHANGELOG.md:19]:
```suggestion
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+- Map canvas follows the gps last position #70
+- Sighting and followers attribute tables are docked in the interface
+- Save status mechanism
+- Audio replay
+- Export database table as csv
+- Merge tools between SAMMO-Boat database
+- Add .ini files to use SAMMO-Boat in operator/admin profile
+- Add keyboard shortcut
+
+
```
Comment (SebastienPeillet) [inline CHANGELOG.md:63]:
```suggestion
```

**Comment by SebastienPeillet** [comment]
Ready to merge for me ! :+1:

**Pull request merged**